flows/src/main/java/com/softwaremill/jox/Flow.java

```diff
@@ -0,0 +1,320 @@
+package com.softwaremill.jox;
```

Comment:
maybe let's put this in a dedicated package, so that there's a package-per-module

Reply:
Done